pycbc/events/stat.py

```diff
@@ -1715,13 +1715,18 @@ def rank_stat_single(self, single_info,
             The array of single detector statistics
         """
         sngls = single_info[1]
-
         ln_noise_rate = sngls['snglstat']
         ln_noise_rate -= self.benchmark_lograte
-        network_sigmasq = sngls['sigmasq']
-        network_logvol = 1.5 * numpy.log(network_sigmasq)
-        benchmark_logvol = sngls['benchmark_logvol']
-        network_logvol -= benchmark_logvol
+        if not np.isnan(sngls['benchmark_logvol']):
+            # Only include this term if the benchmark log volume is defined
```

Comment:
I think the code would be easier to understand with `if np.isnan()` first, as the chunk of comments is only relevant in the live / nan case. ATM you have the comments discussing the edge case just above the code that does the _normal_ calculation but then nothing above the actual edge case itself.